config/min_config.py

```diff
@@ -276,9 +202,68 @@ def build_config(
             q.eta_2,
             q.phi_1,
             q.phi_2,
+            q.m_vis,
+            q.gen_pt_1,
+            q.gen_eta_1,
+            q.gen_phi_1,
+            q.gen_mass_1,
+            q.gen_pdgid_1,
+            q.gen_pt_2,
+            q.gen_eta_2,
+            q.gen_phi_2,
+            q.gen_mass_2,
+            q.gen_pdgid_2,
+            q.gen_m_vis,
+            q.id_wgt_mu_1,
+            q.id_wgt_mu_2,
+            q.iso_wgt_mu_1,
+            q.iso_wgt_mu_2,
         ],
     )
 
+    configuration.add_modification_rule(
+        "global",
+        RemoveProducer(
+            producers=[event.PUweights],
+            samples=["data"],
+        ),
+    )
+    configuration.add_modification_rule(
+        "mm",
+        RemoveProducer(
+            producers=[
+                genparticles.MMGenDiTauPairQuantities,
+                scalefactors.MuonIDIso_SF,
+            ],
+            samples=["data"],
+        ),
+    )
+
+    configuration.add_shift(
+        SystematicShift(
+            name="MuonIDUp",
+            shift_config={"mm": {"muon_sf_varation": "systup"}},
```

Comment:
muon_sf_variation and why "systup" and not just "up"? we always use "up"/"down"

Reply:
this comes from the correction lib json this way ..